### ISSUE TITLE
Very good library, is there an RN version?

### ISSUE THREAD

**Issue by codthing** [gnapse/react-treefold#1]

**Comment by gnapse** [comment]
Hi! By RN I assume you mean React Native, right? I have zero experience with React Native so I'm not sure what this is lacking to work in RN and what needs to be done to make it compatible.

Can you elaborate here on the details (even if obvious for folks with RN experience), or maybe share a link about this topic.

Also, contributions are welcome. I have not used this in a while and I had neglected it since it did not appear to have much of a user base, but I may soon update the dependencies and maybe even port it to TS.

**Comment by codthing** [comment]
@gnapse 
Yes, the ReactNative version. Tree can hardly find a better library in the RN open source community.
